### ISSUE TITLE
Usability: can't close image viewer

### ISSUE THREAD

**Issue by ForestJohnson** [signalapp/Signal-iOS#5889]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Usability: After user opens a picture message, there is a back button. However, if the user zooms in on the image or accidentally brushes the screen in the wrong way, the back button disappears, there's no indication of how to get it back, and users best guess of how to get it back does not work

### Steps to reproduce
- open an image
- zoom or tap image once
- the back button is now gona and there's no indication of how to get it back
- tapping the area where it used to be has no effect
- the user assumed it was broken and actually closed the app and reopened at this point, because that's something they know how to do

**Actual result:** 
User expected to be able to get back button to come back by tapping black region outside the image

**Desired result**
Back button shows up when screen is tapped, including region outside the image

OR

Back button never disappears

### Screenshots
N/A

### Device info
N/A

### Link to debug log
N/A

**Comment by DGrothe-PhD** [comment]
Can confirm this.
Noticed, by the way, a feature:
At least, when zoomed out, swiping left/right lets you view all of your images you shared with that contact, one by one, ordered by date sent.

Workaround to close the image and return to the chat:
1) Zoom out until the image fits the screen again
2) Swipe upwardly from some place inside of the screen.

**Comment by ForestJohnson** [comment]
hi bot

**Comment by ForestJohnson** [comment]
> Back button shows up when screen is tapped, including region outside the image

This is now how it works in signal on android. Thanks for fixing this. Closing.

**Comment by ForestJohnson** [comment]
OOPS  just realized this is the iOS signal repo not the android one 

The bug is still present on iOS, confirmed in the last 60 seconds on my friends phone

**Comment by MarlowBrown** [comment]
Once zoomed out, you are still able to drag the photo down to dismiss it, but I think having that button back is nicer! :)

https://github.com/user-attachments/assets/acacddf8-c42c-4fbb-ac8f-b465f45e6c9d

### Here is the PR for the bug fix! 
[Bug Fix Link here](https://github.com/signalapp/Signal-iOS/pull/5944)

**Comment by DGrothe-PhD** [comment]
> Once zoomed out, you are still able to drag the photo down to dismiss it, but I think having that button back is nicer! :)
> 
> https://github.com/user-attachments/assets/acacddf8-c42c-4fbb-ac8f-b465f45e6c9d
> 
> ### Here is the PR for the bug fix! 
> [Bug Fix Link here](https://github.com/signalapp/Signal-iOS/pull/5944)
> 
> 

I agree. And leave the less experienced user puzzled as they don't know how to close an image right away.

**Comment by sashaweiss-signal** [comment]
Closing as the related PR is now resolved: https://github.com/signalapp/Signal-iOS/pull/5944.
